src/cpp/common/completion_queue_cc.cc

```diff
@@ -20,13 +20,109 @@
 #include <memory>
 
 #include <grpc/grpc.h>
+#include <grpc/support/cpu.h>
 #include <grpc/support/log.h>
 #include <grpcpp/impl/grpc_library.h>
 #include <grpcpp/support/time.h>
 
+#include "src/core/lib/gpr/useful.h"
+#include "src/core/lib/gprpp/manual_constructor.h"
+#include "src/core/lib/gprpp/sync.h"
+#include "src/core/lib/gprpp/thd.h"
+
 namespace grpc {
+namespace {
+
+internal::GrpcLibraryInitializer g_gli_initializer;
```

Comment:
Why is this not static?

Reply:
It's in an anonymous namespace so it has the same effect as static. The [style guide](https://google.github.io/styleguide/cppguide.html#Internal_Linkage) allows either  but we've been mostly using anonymous namespaces recently.